edge/pkg/edged/edged_volumes.go

```diff
@@ -167,8 +172,7 @@ func (e *edged) podVolumesExist(podUID types.UID) bool {
 
 // listProcMounts is come from k8s.io\kubernetes\pkg\util\mount.listProcMounts
 func listProcMounts(mountFilePath string) ([]mount.MountPoint, error) {
```

Comment:
Could we import `ListProcMounts` from https://github.com/kubernetes/utils/blob/master/mount/mount_linux.go#L456?

Reply:
I think those logic is to list all mount point under `/var/lib/edged/xxx/volumes` then umount, the k8s `ListProcMounts` is to list all mount points from `/proc/mounts`, the older logic wrongly use `/var/lib/edged/xxx/volumes` as `ListProcMounts` param and result error logs.

Reply:
Need to think if we need cleanupMoutPoints, why k8s has no this func.

Reply:
K8s check a dir is mount point or not based on `ListProcMounts`, the function is named as `CleanupMountPoint`, refer to https://github.com/kubernetes/utils/blob/master/mount/mount_helper_common.go#L26-L41.